typescript/cli/src/deploy/warp.ts

```diff
@@ -347,9 +347,17 @@ export async function runWarpRouteApply(
       context.registry,
     );
 
+  const { multiProvider } = context;
+  const intermediateOwnerConfig = await promiseObjAll(
```

Comment:
```suggestion
  const { multiProvider } = context;
  // temporarily configure deployer as owner so that warp update after extension
  // can leverage JSON RPC submitter on new chains
  const intermediateOwnerConfig = await promiseObjAll(
```